CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## HEAD (unreleased)
 
+## 3.0 (20th Jan 2023)
+Authors: doits, jstoks, merringtion
+
+* Adds: Support for Ruby 3.0 keyword arguments
+* Removes: Support for Ruby 2.6 and lower
+
```

Comment:
Can you remove the 3.0 heading, thanks.

Is the addition of Ruby 3.0 keyword arguments a breaking change, do those using Ruby 2.x need to make any changes?